vertical-pod-autoscaler/pkg/recommender/model/types.go

```diff
@@ -91,6 +93,12 @@ func ResourcesAsResourceList(resources Resources) apiv1.ResourceList {
 		case ResourceMemory:
 			newKey = apiv1.ResourceMemory
 			quantity = QuantityFromMemoryAmount(resourceAmount)
+			if humanizeMemory && !quantity.IsZero() {
+				rawValues := quantity.Value()
+				humanizedValue := HumanizeMemoryQuantity(rawValues)
+				klog.InfoS("Converting raw value to humanized value", "rawValue", rawValues, "humanizedValue", humanizedValue)
```

Comment:
This may happen quite a lot. Do you mind if we log this at a high verbosity level?

Reply:
sure, which level do you propose?

Reply:
Doing a quick grep of current logs, 4 seems to be a popular choice.
That's my vote
```suggestion
				klog.V(4).InfoS("Converting raw value to humanized value", "rawValue", rawValues, "humanizedValue", humanizedValue)
```